src/style.css

```diff
@@ -1 +1,71 @@
 /* Styles go here */
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --regular-font: #000;
+  --hover-font: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  line-height: 14px;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  flex-wrap: nowrap;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  color: var(--regular-font);
+  cursor: pointer;
```

Comment:
it's default value for `a`

Reply:
I see, thanks for your review. I had to add it because wasn't sure if it's default in all browsers. But now i checked info about compatibility and i know there is no need in writing default value.